ui/packages/catalog-ui-search/src/main/webapp/js/model/Alert.js

```diff
@@ -29,7 +29,7 @@ module.exports = Backbone.AssociatedModel.extend({
       this.set('id', Common.generateUUID())
     }
   },
-  getTimeComparator() {
+  getTime() {
```

Comment:
❗ Notifications are also created when users upload data from the UI. They rely on the same `getTimeComparator` method, so I recommend we revert this rename https://github.com/codice/ddf/blob/9c528c54f2540d7fb01e271fb83a00942ed20fea/ui/packages/catalog-ui-search/src/main/webapp/js/model/UploadBatch.js#L189-L191

Reply:
They share a name but are different functions; the one in `UploadBatch.js` is comparing by `sentAt`, but this change compares by `when` so there shouldn't be any impact.

Besides, the name is totally erroneous: these do not return comparator functions, they only return values. I would prefer renaming the one in UploadBatch too;

Reply:
Yes they use different values but the idea is the same - some date/time used for sorting purposes. I like that `Comparator` in the name implies that it is used somehow in sorting them, but I also agree that the name doesn't match what the method actually does so I'm on the fence. I'll leave it to your discretion

Reply:
<img width="430" alt="Screen Shot 2020-09-22 at 4 27 45 PM" src="https://user-images.githubusercontent.com/6551038/93943997-e99db980-fcf0-11ea-8d56-738da6bea5dc.png">
this rename seems to be causing other issues during the hero

Reply:
Reverted the rename, kept only the call to `sort()`